base/inc/AdePT/BVHNavigator.h

```diff
@@ -211,17 +211,18 @@ class BVHNavigator {
   // Relocate a state that was returned from ComputeStepAndNextVolume: It first
   // removes all volumes from the state that were left, and then recursively
   // locates the pushed point in the containing volume.
-  __host__ __device__ static void RelocateToNextVolume(vecgeom::Vector3D<vecgeom::Precision> const &globalpoint,
+  __host__ __device__ static void RelocateToNextVolume(vecgeom::Vector3D<vecgeom::Precision> &globalpoint,
                                                        vecgeom::Vector3D<vecgeom::Precision> const &globaldir,
                                                        vecgeom::NavStateIndex &state)
   {
     // Push the point inside the next volume.
-    vecgeom::Vector3D<vecgeom::Precision> pushed = globalpoint + 1.E-6 * globaldir;
+    static constexpr double kPush = 1.e-8;
```

Comment:
This should read something like 10*vecgeom::kTolerance to make it vecgeom::Precision proof.

Reply:
Maybe, but for now this PR is only moving the relevant code and it wasn't there before, so I'd prefer not to do this in the current PR.